src/ModuleHomomorphism.jl

```diff
@@ -12,6 +12,8 @@
 
 mat(f::Map(FPModuleHomomorphism)) = f.matrix
```

Comment:
Maybe replace this by a deprecation instead? So we do not forget to remove it sometime in the future?